Wikipedia/UI-V5/WMFArticleSectionCell.m

```diff
@@ -6,37 +6,15 @@
 
 @interface WMFArticleSectionCell ()
 
-@property (weak, nonatomic) IBOutlet NSLayoutConstraint* leadingIndentationConstraint;
+@property (weak, nonatomic) IBOutlet UIButton* disclosureIndicatorButton;
 
 @end
 
 @implementation WMFArticleSectionCell
 
-- (instancetype)initWithCoder:(NSCoder*)coder {
-    self = [super initWithCoder:coder];
-    if (self) {
-        [self setSelectionBackgroundColor:[UIColor colorWithWhite:0.0 alpha:0.1]];
-        [self wmf_makeCellDividerBeEdgeToEdge];
-    }
-    return self;
-}
-
-- (void)setSelectionBackgroundColor:(UIColor*)color {
-    UIView* selectionColor = [[UIView alloc] init];
-    selectionColor.backgroundColor = color;
-    self.selectedBackgroundView    = selectionColor;
-}
-
-- (void)setLevel:(NSNumber*)level {
-    [self applyIndentForLevel:level];
-}
-
-- (void)applyIndentForLevel:(NSNumber*)level {
```

Comment:
FYI - when you re-indent… there are some properties on UITableViewCell you can use instead of constraints (if you prefer):

```
indentationLevel
indentationWidth
```